traffic_portal/app/src/common/api/DeliveryServiceSslKeysService.js

```diff
@@ -47,7 +43,11 @@ var DeliveryServiceSslKeysService = function($http, locationUtils, messageModel,
 
         return $http.post(ENV.api['root'] + endpoint, generateSslKeyForm).then(
             function(result) {
-            	messageModel.setMessages([{level: 'success', text: message + deliveryService.xmlId}], true);
+            	if (message === null) {
+            		messageModel.setMessages(result.data.alerts, true);
+				} else {
+					messageModel.setMessages([{level: 'success', text: message + deliveryService.xmlId}], true);
+				}
```

Comment:
Indentation here mixes spaces and tabs - unfortunately the file currently appears to use spaces, so we should stick with that inferior choice.